.github/workflows/ci.yml

```diff
@@ -10,5 +10,5 @@ jobs:
         with:
           node-version: 14.x
 
-      - run: npm install
+      - run: npm install --ignore-scripts
```

Comment:
During npm install, the Lighthouse package which is installed from GitHub
(@master branch) triggers the newly added `prepack` script [1], but due to
a bug in NPM [2] the devDependencies are not available when the script
is run, thus making the installation fail.

[1]: GoogleChrome/lighthouse#13261
[2]: npm/cli#1865